src/presentation/http/router/noteSettings.ts

```diff
@@ -214,6 +236,10 @@ const NoteSettingsRouter: FastifyPluginCallback<NoteSettingsRouterOptions> = (fa
           $ref: 'NoteSchema#/properties/id',
         },
       },
+      body: {
+        // since notsettings fields are optional
+        properties: {},
```

Comment:
it should validate properties and their types. You need to describe all available properties